<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect</class>
 <widget class="QDialog" name="connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="connect_body" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>651</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget #connect_body
{
	background-color: rgb(56, 58, 89);
	color: rgb(220, 220, 220);
	border-radius: 10px;
}

#label_ci_title
{
	color: rgb(216, 169, 255);
}

#toolButton_close
{
	background-color: transparent;
	background-image: url(:/resources/icons8-close-32.png);
}

#lineEdit
{
background-color: transparent;
	border-top: none;
	border-left: none;
	border-right: none;
	border-bottom: 2px solid rgb(35, 35, 52);
	color: rgb(216, 189, 255);
}

#lineEdit:focus
{
background-color:  rgb(35, 35, 52);
	border-top: none;
	border-left: none;
	border-right: none;
	border-bottom: 2px solid rgb(35, 35, 52);
}

#lineEdit_2
{
background-color: transparent;
	border-top: none;
	border-left: none;
	border-right: none;
	border-bottom: 2px solid rgb(35, 35, 52);
	color: rgb(216, 189, 255);

}

#lineEdit_2:focus
{
background-color:  rgb(35, 35, 52);
	border-top: none;
	border-left: none;
	border-right: none;
	border-bottom: 2px solid rgb(35, 35, 52);
}

QLabel{		color: rgb(216, 169, 255); }
QCheckBox {		color: rgb(216, 169, 255); }

#toolButton
{
	background-color: transparent;
	
	background-color: rgb(35, 35, 52);
color: rgb(216, 169, 255); 
border-radius: 3px;
}
</string>
   </property>
   <widget class="QLabel" name="label_ci_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;strong&gt;CI&lt;/strong&gt; Tools</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_close">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Show password</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Remeber me</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>If you don't have an account, please click register</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>v1.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_close</sender>
   <signal>clicked()</signal>
   <receiver>connect</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
